src/graphic/style_manager.cc

```diff
@@ -307,7 +307,7 @@ const RGBColor& StyleManager::minimap_icon_frame() const {
 }
 
 std::string StyleManager::color_tag(const std::string& text, const RGBColor& color) {
-	static boost::format f("<font color=%s>%s</font>");
+	boost::format f("<font color=%s>%s</font>");
```

Comment:
// must not be static to be threadsafe

-- we will have lots of such code, uuhm